src/common/types/store-data/unified-data-interface.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// this is similar to `TestEngine` interface from axe-core
+export interface ScanEngine {
```

Comment:
This name doesn't indicate what type of object this is. For reference, OsData indicates that the interface holds data about the OS. Personally, I prefer "properties", but that's up to you.
This is important because without the distinction, it would be easy to look at this interface and expect to see functions which operate the scan engine.

Reply:
Updated to say "properties"